action.yml

```diff
@@ -8,16 +8,20 @@ branding:
   color: orange
 
 inputs:
-  USER_TOKEN: 
+  USER_ACCESS_TOKEN: 
```

Comment:
Should we use snake_case for these inputs instead of the upper case? :thinking_face:
I saw other actions prefer using snake_case.
And we just need to change here, other places still use the UPPER_CASE as Github actions will transform these values for us

Reply:
Thanks for the suggestion @hoangmirs! Updated in fa4bbaf1e6006b0cd93d0d3efcdfc822b8470898 👍